pkg/tfgen/generate.go

```diff
@@ -203,6 +208,7 @@ type variable struct {
 	name   string
 	out    bool
 	opt    bool
+	config bool
```

Comment:
None of the rest have comments - but this would have been a lot easier to follow if we commented here what `config = true` means.

Reply:
True--I will add a comment.